README.md

```diff
@@ -109,16 +109,21 @@ Attempts to authenticate with Face ID/Touch ID.
 Returns a `Promise` object.
 
 __Arguments__
-- `reason` - An _optional_ `String` that provides a clear reason for requesting authentication.
+- `reason` - **optional** - `String` that provides a clear reason for requesting authentication.
+
+- `config` - **optional** - configuration object for more detailed dialog setup:
+  - `title` - **Android** - title of confirmation dialog
+  - `color` - **Android** - color of confirmation dialog
+  - `failbackLabel` - **iOS** - by default specified 'Show Password' label. If set to empty string label is invisible.
```

Comment:
here you call it `failbackLabel`, but in the native module you're looking for `fallbackLabel`. Could you replace the `failbackLabel` with `fallbackLabel` in all instances.

Thanks